.github/workflows/ci.yml

```diff
@@ -101,7 +101,7 @@ jobs:
         run: pnpm i
 
       - name: Install Playwright Dependencies
-        run: pnpm exec playwright install chromium --with-deps
+        run: pnpm exec playwright install chromium --with-deps --only-shell
```

Comment:
is this just for vitest CI or we, as vitest users can already opt to use new chromium binary, the one that is close to real chrome and not this old one (shell)

Reply:
`--only-shell` is the old one. If you want to adopt a new one, you can follow the playwright documentation and specify a `channel` in `providerOptions.launch`